docs/src/api/class-apirequest.md

```diff
@@ -12,26 +12,26 @@ see [APIRequestContext].
 
 Creates new instances of [APIRequestContext].
 
-### option: APIRequest.newContext.useragent = %%-context-option-useragent-%%
+### option: `APIRequest.newContext.useragent = %%-context-option-useragent-%%`
```

Comment:
Playwright generates code from the documentation, so we can't change sections in the md files.